### ISSUE TITLE
Icon request: Behance

### ISSUE THREAD

**Issue by kuroikyu** [feathericons/feather#400]
I'd like to request an icon for Behance:
- Icon name: Behance.
- At least one use case: Sharing the user's Behance profile.
- Screenshots of similar icons: [FontAwesome1](https://fontawesome.com/icons/behance?style=brands) and [FontAwesome2](https://fontawesome.com/icons/behance-square?style=brands).

**Comment by MarcelloTheArcane** [comment]
The `e` looks a bit comic-sans-y, so if anyone could tidy it up.. 😁 

![image](https://user-images.githubusercontent.com/21159570/61550202-c7077500-aa49-11e9-8dd2-b6a0bcc26c7f.png)

![image](https://user-images.githubusercontent.com/21159570/61550220-d38bcd80-aa49-11e9-9f1c-1c3e99c76d8e.png)

**Comment by locness3** [comment]
I don't know what I can do for this one. @ahtohbi4 @johnletey ? Please provide your Figma link also @MarcelloTheArcane

**Comment by MarcelloTheArcane** [comment]
Letters + 24px resolution = problems 😏

[Chez icons](https://www.figma.com/file/kbBrVCZ84E8ZazGrBxzyTc/Feather-Component-Library-Copy) (down the bottom)

**Comment by johnletey** [comment]
@locness3 How does these look? [Figma](https://www.figma.com/file/C7cEHvYfSsGpvx9utbnFnhJy/Feather-Icons?node-id=61%3A2)

![Shot 2019-07-21 at 08 20 19](https://user-images.githubusercontent.com/30328854/61588273-68004800-ab90-11e9-824d-3bc10a2ab688.jpg)

The bottom-right of the "e" still looks weird though ... have any thoughts @ahtohbi4?

**Comment by locness3** [comment]
A bit better than the one from @MarcelloTheArcane

**Comment by johnletey** [comment]
Great ... should I submit?

**Comment by locness3** [comment]
idk

**Comment by MarcelloTheArcane** [comment]
Yep, that's neater. Don't know if you could somehow increase the gap between the circle and the tail of the `e`?

**Comment by johnletey** [comment]
@MarcelloTheArcane I'll see what I can do!

**Comment by charlieclark** [comment]
@johnletey this looks pretty good to me! I wanted to use it in my project and I tried exporting the svg from your figma file but was getting some weirdness with the B... The path looks fine in figma but once it exports it converts it into a shape... The e looks fine tho

**Comment by locness3** [comment]
@charlieclark see #763

**Comment by AlvarezLopezAlberto** [comment]
I try to make a Behance icon 
![BehanceIcon](https://user-images.githubusercontent.com/30964090/123846367-e3b9b200-d8da-11eb-9d06-689ba76459bf.png)

**Comment by locness3** [comment]
ok
